src/server/scripts/Northrend/IcecrownCitadel/icecrown_citadel.h

```diff
@@ -147,6 +150,8 @@ enum ICCreaturesIds
     NPC_MURADIN_BRONZEBEARD_QUEST               = 38607,
     NPC_UTHER_THE_LIGHTBRINGER_QUEST            = 38608,
     NPC_LADY_SYLVANAS_WINDRUNNER_QUEST          = 38609,
+    NPC_NERUBAR_BROODKEEPER                     = 36725,
+    NPC_NERUBAR_BROODKEEPER_H                   = 38058,
```

Comment:
Do not use heroic entry, heroic entry is never spawned in game

Reply:
Done